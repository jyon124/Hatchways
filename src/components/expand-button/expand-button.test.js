describe("Student Expand Button Component Tests", () => {
    it.todo("Renders a Student Expand Button component");
    it.todo("Test if Toggle function invokes when its clicked");
});
