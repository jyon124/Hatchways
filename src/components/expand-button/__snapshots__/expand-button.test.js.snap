// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Expand Button Component Tests Renders a Student Expand Button component 1`] = `"<div class=\\"expandBtn\\"><div class=\\"horizontalLine\\"></div><div></div></div>"`;
